copilot/fastapi/tests/conftest.py

```
@@ -0,0 +1,62 @@
+    """Fixture to mock Git subprocess commands"""
+    with patch("subprocess.run") as mock_run:
+        mock_run.return_value = MagicMock(returncode=0)
+        yield mock_run
+
+@pytest.fixture
+def mock_yaml_operations():
+    """Fixture to mock YAML operations"""
+    with patch("yaml.safe_load") as mock_yaml_load:
+        mock_yaml_load.return_value = {
+            "image": "test-image",
+            "tag": "latest"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)